com.microsoft.mrtk.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -71,7 +71,7 @@ public static InteractionModeManager Instance
         public void InitializeControllers()
         {
             controllerMapping.Clear();
-            foreach (XRController xrController in FindObjectsOfType<XRController>())
+            foreach (XRController xrController in ComponentCache<XRController>.FindInstanceList())
```

Comment:
This function should not cache stuff, as its only called from the Unity Editor's inspector window.